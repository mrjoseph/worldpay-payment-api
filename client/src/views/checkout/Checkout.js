import React, { Component } from 'react';
import { connect } from 'react-redux';
import { Link } from "react-router-dom";
class Checkout extends Component {
  render() {

    return (
      <div>
        <Link to="/payments">Proceed to checkout</Link>
      </div>
    );
  }
}

const mapStateToProps = () => ({});
const mapDispatchToProps = () => ({});
export default connect(mapStateToProps,mapDispatchToProps)(Checkout);
