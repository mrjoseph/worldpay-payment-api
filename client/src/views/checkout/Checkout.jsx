import React from 'react';
import { Link, withRouter } from 'react-router-dom';

const Checkout = () => <div><Link to="/payment">Proceed to checkout</Link></div>;

export default withRouter(Checkout);
