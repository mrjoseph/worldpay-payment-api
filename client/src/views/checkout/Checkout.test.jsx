import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Checkout from './Checkout';

describe('Header', () => {
  it('Should render the checkout view without props', () => {
    const rendered = renderer.create(<Router><Checkout /></Router>);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
