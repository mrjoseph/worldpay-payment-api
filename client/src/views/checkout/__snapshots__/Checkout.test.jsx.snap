// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render the checkout view without props 1`] = `
<div>
  <a
    href="/payment"
    onClick={[Function]}
  >
    Proceed to checkout
  </a>
</div>
`;
